spec/mailers/user_mailer_spec.rb

```diff
@@ -40,17 +40,23 @@
 
   describe 'reservation_status_update' do
     it 'sends to the reserver' do
+      # make it a request so the email sends
+      @res.update_attributes(flags: Reservation::FLAGS[:request])
```

Comment:
But I think that there are more cases where we don't want the reservation to be a request.

Perhaps a better solution is to change the mailer call to force a certain email type?